### DISCUSSION TITLE
TE3 not saving connection - model refresh not possible

### DISCUSSION THREAD

**Discussion by LazareSycamore** [TabularEditor/TabularEditor3#752] [🙋♀️TE3 Q&A]
Hi,

I've recently started having problems with model refresh. Whenever I try to refresh entire model or a single table, I receive error "A connection could not be made...". At the same time I am able to do this via SSMS. After some further testing and investigation, I've observed few things:
1. Connection string settings are not being saved after modification in TE3.
2. I go to SSMS, enter credentials and refresh model without problem.
3. I return to TE3 and again, I cannot refresh the model.
4. I go back to SSMS and not only my credentials are wiped out, but also entire connection string settings. 

Any idea, why this is happening? I've got impression, that this had started happening after I've updated my TE3 to 
![image](https://user-images.githubusercontent.com/45509889/206644808-68a5ccf0-130e-4059-9d1e-8dc7edfb4d33.png)

**Comment by otykier** [comment]
Could you try to update to the latest version [3.4.0](https://docs.tabulareditor.com/te3/other/release-notes/3_4_0.html?tabs=downloads) and let us know if the issue persists? Thanks.
